charts/seed-bootstrap/charts/fluent-bit/templates/_helpers.tpl

```diff
@@ -73,6 +73,13 @@ filter-kubernetes.conf: |-
       Parser              kubernetesdashboardParser
       Reserve_Data        True
 
+  [FILTER]
+      Name                parser
+      Match               kubernetes.*shoot-dns-service*shoot-dns-service*
+      Key_Name            log
+      Parser              shootDnsServiceParser
+      Reserve_Data        True
```

Comment:
This should not be added here since it's related to an extension. Use https://github.com/gardener/gardener/blob/master/docs/extensions/logging-and-monitoring.md#fluent-bit-log-parsers-and-filters instead.